tarodb/assets_store.go

```diff
@@ -762,6 +801,54 @@ func (a *AssetStore) QueryAssetBalancesByGroup(ctx context.Context,
 	return balances, nil
 }
 
+// FetchGroupedAssets fetches the set of assets with non-nil group keys.
+func (a *AssetStore) FetchGroupedAssets(ctx context.Context) (
+	[]*AssetHumanReadable, error) {
+
+	var (
+		dbAssets []RawGroupedAsset
+		err      error
+	)
+
+	readOpts := NewAssetStoreReadTx()
+	dbErr := a.db.ExecTx(ctx, &readOpts, func(q ActiveAssetsStore) error {
+		dbAssets, err = q.FetchGroupedAssets(ctx)
+		return err
+	})
+	if dbErr != nil {
+		return nil, dbErr
+	}
+
+	groupedAssets := make([]*AssetHumanReadable, len(dbAssets))
+	for i, a := range dbAssets {
+		amount := uint64(a.Amount)
+		locktime := extractSqlInt32[uint64](a.LockTime)
```

Comment:
micro nit: `lockTime` and `relativeLockTime`? Flagged by the grammar checker of my IDE :sweat_smile: